go/worker/compute/executor/committee/node.go

```diff
@@ -35,86 +36,26 @@ import (
 )
 
 var (
-	errSeenNewerBlock     = fmt.Errorf("executor: seen newer block")
-	errRuntimeAborted     = fmt.Errorf("executor: runtime aborted batch processing")
-	errBatchTooLarge      = p2pError.Permanent(fmt.Errorf("executor: batch too large"))
 	errMsgFromNonTxnSched = fmt.Errorf("executor: received txn scheduler dispatch msg from non-txn scheduler")
 
-	// proposeTimeoutDelay is the duration to wait before submitting the propose timeout request.
-	proposeTimeoutDelay = 2 * time.Second
+	// priorityDelay is the duration to wait before accepting proposal
+	// from the next backup scheduler.
+	priorityDelay = 2 * time.Second
```

Comment:
Wondering if this should be a parameter in the runtime descriptor (even though it can't be enforced, but just to maybe make coordination easier)?

Reply:
It could be. The only question is what to do if it is empty, do we use a default value?

Reply:
That sounds reasonable.